--- PKGBUILD	2025-04-22 12:49:13.505249324 -0300
+++ PKGBUILD.backup	2025-04-22 12:50:59.173968664 -0300
@@ -28,7 +28,7 @@
 )
 makedepends=(
 	'alsa-lib' 'cmake' 'git' 'libevdev' 'libminiupnpc.so' 'libpulse'
-	'libudev.so' 'ninja' 'python' 'qt6-base' 'qt6-svg'
+	'libudev.so' 'python' 'qt6-base' 'qt6-svg'
 )
 checkdepends=('gtest')
 optdepends=('pulseaudio: PulseAudio backend')
@@ -58,6 +58,7 @@
         '7db29101fc7496355776eee0701ddb971147aea096828f73dc02501d8981a8f1105f16e206a24f3ab94d169dc7ea0443c37b664c25ba064533b7cdcc644bd6f4'
         'd9e6ba73de8e1c49a7ebf9efe6caffcffbe1a545dfb61caebe2b830d8f496aaa221269c25a3f849ba02228dfb866b362c8c74f7e897e66a9362469dea679721d')
 
+
 _sourcedirectory="$pkgbase"
 
 prepare() {
@@ -108,16 +109,25 @@
 	# CMAKE_SKIP_RPATH - do not add run time path information (the package in the repos does it, presumably because of reproducible builds)
 	# USE_SYSTEM_LIBS - we want to use system libs where possible
 	# USE_SYSTEM_LIBMGBA - the current version of mgba in the repos is not compatible with Dolphin
-	cmake -S '.' -B 'build/' -G Ninja \
+	cmake -S . -B build \
 		-DCMAKE_BUILD_TYPE=None \
 		-DCMAKE_SKIP_RPATH=ON \
 		-DCMAKE_INSTALL_PREFIX='/usr' \
 		-DDISTRIBUTOR='aur.archlinux.org/packages/dolphin-emu-git' \
+		-DENABLE_ANALYTICS=OFF \
 		-DENABLE_AUTOUPDATE=OFF \
+		-DENABLE_HEADLESS=OFF \
+		-DENABLE_LTO=ON \
+		-DENABLE_TESTS=OFF \
+		-DENABLE_VULKAN=OFF \
+		-DUSE_DISCORD_PRESENCE=OFF \
+		-DUSE_MGBA=OFF \
+		-DUSE_RETRO_ACHIEVEMENTS=ON \
 		-DUSE_SYSTEM_LIBS=ON \
 		-DUSE_SYSTEM_LIBMGBA=OFF \
+		-DUSE_UPNP=OFF \
 		-Wno-dev
-	cmake --build 'build/'
+	cmake --build build
 }
 
 check() {
@@ -125,10 +135,6 @@
 	cd "$srcdir/$_sourcedirectory/"
 	_checkversion="$(git describe | cut --delimiter='-' --fields=1-2)"
 
-	# Run tests
-	cd "$srcdir/$_sourcedirectory/build/"
-	ninja unittests
-
 	# Verify that the basic functionality works
 	_checkoutput="$(QT_QPA_PLATFORM='offscreen' "$srcdir/$_sourcedirectory/build/Binaries/$_noguipkgname" --version)"
 	printf '%s\n' "$_checkoutput"
