--- PKGBUILD
+++ PKGBUILD.backup
@@ -1,3 +1,5 @@
+# Patched AUR PKGBUILD
+
 # Maintainer: Daniel Peukert <daniel@peukert.cc>
 # Contributor: Maxime Gauduin <alucryd@archlinux.org>
 # Contributor: Lightning <sgsdxzy@gmail.com>
@@ -20,19 +22,18 @@
 	# Based on the repo package
 	'bluez-libs' 'bzip2' 'enet' 'gcc-libs' 'glibc' 'hidapi' 'libavcodec.so'
 	'libavformat.so' 'libavutil.so' 'libcurl.so' 'libfmt.so' 'libgl'
-	'libsfml-network.so' 'libsfml-system.so' 'libspng.so' 'libswscale.so'
+	'libsfml-network.so' 'libspng.so' 'libswscale.so'
 	'libusb-1.0.so' 'libx11' 'libxi' 'libxrandr' 'lz4' 'lzo' 'mbedtls2' 'pugixml'
 	'sdl2' 'sfml' 'speexdsp' 'xxhash' 'xz' 'zstd'
 	# Additional dependencies to replace vendored deps
 	'cubeb' 'minizip-ng'
 )
 makedepends=(
-	'alsa-lib' 'cmake' 'git' 'libevdev' 'libminiupnpc.so' 'libpulse'
-	'libudev.so' 'ninja' 'python' 'qt6-base' 'qt6-svg'
+	'alsa-lib' 'cmake' 'git' 'libevdev' 'libpulse'
+	'libudev.so' 'python' 'qt6-base' 'qt6-svg'
 )
 checkdepends=('gtest')
 optdepends=('pulseaudio: PulseAudio backend')
-options=('!lto')
 source=(
 	"$pkgbase::git+https://github.com/$_mainpkgname/$_projectname"
 	"$pkgbase-implot::git+https://github.com/epezent/implot.git"
@@ -58,6 +59,7 @@
         '7db29101fc7496355776eee0701ddb971147aea096828f73dc02501d8981a8f1105f16e206a24f3ab94d169dc7ea0443c37b664c25ba064533b7cdcc644bd6f4'
         'd9e6ba73de8e1c49a7ebf9efe6caffcffbe1a545dfb61caebe2b830d8f496aaa221269c25a3f849ba02228dfb866b362c8c74f7e897e66a9362469dea679721d')
 
+
 _sourcedirectory="$pkgbase"
 
 prepare() {
@@ -108,16 +110,25 @@
 	# CMAKE_SKIP_RPATH - do not add run time path information (the package in the repos does it, presumably because of reproducible builds)
 	# USE_SYSTEM_LIBS - we want to use system libs where possible
 	# USE_SYSTEM_LIBMGBA - the current version of mgba in the repos is not compatible with Dolphin
-	cmake -S '.' -B 'build/' -G Ninja \
+	cmake -S . -B build \
 		-DCMAKE_BUILD_TYPE=None \
 		-DCMAKE_SKIP_RPATH=ON \
 		-DCMAKE_INSTALL_PREFIX='/usr' \
 		-DDISTRIBUTOR='aur.archlinux.org/packages/dolphin-emu-git' \
+		-DENABLE_ANALYTICS=OFF \
 		-DENABLE_AUTOUPDATE=OFF \
+		-DENABLE_HEADLESS=OFF \
+		-DENABLE_LTO=ON \
+		-DENABLE_TESTS=OFF \
+		-DENABLE_VULKAN=OFF \
+		-DUSE_DISCORD_PRESENCE=OFF \
+		-DUSE_MGBA=OFF \
+		-DUSE_RETRO_ACHIEVEMENTS=ON \
 		-DUSE_SYSTEM_LIBS=ON \
 		-DUSE_SYSTEM_LIBMGBA=OFF \
+		-DUSE_UPNP=OFF \
 		-Wno-dev
-	cmake --build 'build/'
+	cmake --build build
 }
 
 check() {
@@ -125,10 +136,6 @@
 	cd "$srcdir/$_sourcedirectory/"
 	_checkversion="$(git describe | cut --delimiter='-' --fields=1-2)"
 
-	# Run tests
-	cd "$srcdir/$_sourcedirectory/build/"
-	ninja unittests
-
 	# Verify that the basic functionality works
 	_checkoutput="$(QT_QPA_PLATFORM='offscreen' "$srcdir/$_sourcedirectory/build/Binaries/$_noguipkgname" --version)"
 	printf '%s\n' "$_checkoutput"
@@ -138,7 +145,7 @@
 package_dolphin-emu-git() {
 	pkgdesc="$pkgdesc$_pkgdescappend"
 	depends+=(
-		'alsa-lib' 'hicolor-icon-theme' 'libevdev' 'libminiupnpc.so' 'libpulse'
+		'alsa-lib' 'hicolor-icon-theme' 'libevdev' 'libpulse'
 		'libudev.so' 'qt6-base' 'qt6-svg'
 	)
 	provides=("$_mainpkgname")
