--- PKGBUILD	2025-04-19 01:20:22.584024989 -0300
+++ PKGBUILD.backup	2025-04-22 15:57:24.554694426 -0300
@@ -1,3 +1,5 @@
+# Patched AUR PKGBUILD
+
 # Maintainer: Andreas Radke <andyrtr@archlinux.org>
 # Maintainer: Robin Candau <antiz@archlinux.org>
 # Contributor: Tom Gundersen <teg@jklm.no>
@@ -11,7 +13,7 @@
 url="http://www.bluez.org/"
 arch=('x86_64')
 license=('GPL-2.0-only')
-makedepends=('dbus' 'libical' 'systemd' 'alsa-lib' 'json-c' 'ell' 'python-docutils' 'python-pygments'      )
+makedepends=('dbus' 'libical' 'alsa-lib' 'json-c' 'ell' 'python-docutils' 'python-pygments'      )
 source=(https://www.kernel.org/pub/linux/bluetooth/bluez-${pkgver}.tar.xz fake-ps3.patch
         bluetooth.modprobe)
 # see https://www.kernel.org/pub/linux/bluetooth/sha256sums.asc
@@ -34,6 +36,7 @@
           --localstatedir=/var \
           --libexecdir=/usr/lib \
           --with-dbusconfdir=/usr/share \
+          --disable-systemd \
           --enable-btpclient \
           --enable-midi \
           --enable-sixaxis \
